src/LocalDate.php

```diff
@@ -456,6 +456,12 @@ public function plusDays(int $days): LocalDate
             return $this;
         }
 
+        if ($days === 1) {
+            return $this->day >= 28 && $this->day === $this->getLengthOfMonth()
```

Comment:
`$this->day >= 28` is redundant with the second condition, I guess this is another optimization to avoid calling `getLengthOfMonth()` needlessly?

Reply:
It is indeed. 28 times out of 31, we can prevent a method call with a simple math condition. Seemed like a good deal. ;)